src/diptest.py

```diff
@@ -301,12 +291,558 @@ def dip_pval_tabinterpol(dip, N):
         N       -   number of observations
     '''
 
-    if qDiptab_df is None:
-        raise DataError("Tabulated p-values not available. See installation instructions.")
-
     if np.isnan(N) or N < 10:
         return np.nan
 
+    qDiptab_dict = {'0': {4: 0.125,
```

Comment:
I think qDiptab should be put in a separate csv-file instead of directly in the python module. It will be much easier to overview both the data and the code when they are separated.